### ISSUE TITLE
Replace menu & drawer with standard drop-down menu

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdropcms.org#305]
The current menu & drawer situation doesn't work well on mobile. We now have a nice mobile drop-down menu solution in core. Should we considder switching our navigation on backdropcms.org to use a mobile-friendly drop-down menu?

**Comment by docwilmot** [comment]
Yes, yes, yes, yes and especially yes! [I never did like the drawer](https://github.com/backdrop-ops/backdropcms.org/issues/38#issuecomment-75047587)

**Comment by klonos** [comment]
I [also never liked the drawer](https://github.com/backdrop-ops/backdropcms.org/issues/38#issuecomment-75066028) 😄 ... 👍 to all points.

**Comment by Graham-72** [comment]
Yes, please make the change.

**Comment by klonos** [comment]
This is done. Any remaining tasks?

**Comment by jenlampton** [comment]
Yes, we need to add a drop-down menu and style it accordingly.
